arangod/Agency/Agent.cpp

```diff
@@ -1359,8 +1358,9 @@ void Agent::lead() {
 }
 
 // When did we take on leader ship?
-SteadyTimePoint const& Agent::leaderSince() const {
-  return _leaderSince;
+long long Agent::leaderSince() const {
```

Comment:
We should not use `long long` here, but rather `int64_t`. Furthermore, `std::chrono::seconds` only guarantees 35bits, which is OK with an epoch since 1070, but otherwise too small, we should not use this. Either make the return value a `std::chrono::duration<int64_t>` (i.e. seconds as signed 64bit number) or directly `int64_t` in seconds, and document this.

Reply:
Done